exercises/pascals-triangle/example.rs

```diff
@@ -0,0 +1,25 @@
+pub struct PascalsTriangle {
+    row_count: u32,
+}
+
+impl PascalsTriangle {
+    pub fn new(row_count: u32) -> Self {
+        PascalsTriangle { row_count: row_count }
+    }
+
+    pub fn rows(&self) -> Vec<Vec<u32>> {
+        (0..self.row_count).map(|row| PascalsTriangle::row(row)).collect()
+    }
+
+    pub fn row(number: u32) -> Vec<u32> {
+        let mut r = vec![1];
+
+        for p in 1..(number + 1) {
+            if let Some(last) = r.pop() {
+                let next = last as f32 * ((number as f32 + 1f32 - p as f32) / p as f32);
```

Comment:
Someone may then righfully say  "(a * b) / c leaves us more susceptible to integer overflow" though, which I can't deny! Even using u64 will only stave that off for so long.